Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -1291,6 +1336,85 @@ def process_search_get_command(client: Client, args: Dict) -> List[CommandResult
     return job_result_list
 
 
+def add_threat_tags_command(client: Client, args: Dict) -> CommandResults:
+    tags = args.get("tags")
+    threat_id = args.get("threat_id")
+    result = client.update_threat_tags(threat_id, tags)
+
+    readable_output = tableToMarkdown(f'Successfully updated threat: "{threat_id}" with tags "{tags}"', result, removeNull=True)
```

Comment:
```suggestion
    readable_output = tableToMarkdown(f'Successfully updated threat: "{threat_id}"', result, removeNull=True)
```
Isn't it unnecessary? because it will appear in the table?